apps/dashboard/src/pages/subscribers.tsx

```diff
@@ -17,6 +18,7 @@ export const SubscribersPage = () => {
       <PageMeta title="Subscribers" />
       <DashboardLayout headerStartItems={<h1 className="text-foreground-950 flex items-center gap-1">Subscribers</h1>}>
         <SubscriberList className="px-2.5" />
+        <Outlet />
```

Comment:
please use the `AnimatedOutlet` instead as without it the close animation will not work